src/include/network/lan/cmdline.rb

```diff
@@ -134,8 +137,8 @@ def ShowHandler(options)
           if key == Ops.get(options, "id", "0")
             # create plain text from formated HTML
             text = Builtins.sformat(
-              "echo \"%1\"|sed s/'<br>'/'\\n'/g|sed s/'<\\/li>'/'\\n'/g|sed s/'<[/a-z]*>'/''/g",
-              Ops.get_string(value, "rich_descr", "")
+              "/usr/bin/echo %1 | /usr/bin/sed 's/<br>/\\n/g;s/<\\/li>/\\n/g;s/<[/a-z]*>//g'",
```

Comment:
1. `echo` is a shell builtin. Replacing it with `/usr/bin/echo` may produce slightly different results
2. but anyway, we should just use String#gsub, it will be easier than making the shell+sed magic safe
3. a unit test for "create plain text from formated HTML" would be great. Yes, this PR is getting out of hand already :D

Reply:
@mvidner @mchf done at https://github.com/yast/yast-network/pull/709/commits/0bdd17af6bc8e762e2795700d74786bd9ba7a6e8